src/azure-cli/HISTORY.rst

```diff
@@ -113,6 +113,8 @@ az webapp create : Fixing issue when running the command with --runtime
 * Each cluster gets a separate service principal to improve isolation
 * Add --nodepool-tags to node pool when creating azure kubernetes cluster
 * Add --tags when adding or updating a nodepool to cluster
+* Add --nodepool-labels when creating azure kubernetes cluster
+* Add --labels when adding a nodepool to cluster
```

Comment:
Please remove the history notes here, it's in a previous version 2.0.81. You've already added it in the PR description. Our release script will fetch the history notes in PR title or PR description and insert into HISTORY.rst before release. You no longer need to modify HISTORY.rst in a regular PR.

Reply:
Nice. Good to know!